tasks/Logger.cpp

```diff
@@ -61,46 +63,15 @@ Logger::~Logger()
 
 bool Logger::startHook()
```

Comment:
The method is missing a call to the parent method, please add it (it was already the case, but let's add it for good measure)

~~~
if (! LoggerBase::startHook()) {
  return false;
}
~~~